website/content/en/blog/product-thinking-for-platforms.md

```diff
@@ -0,0 +1,134 @@
+
+If we're talking about platform engineering, we of course need to bring up the [CNCF Platform Engineering Maturity Model](https://tag-app-delivery.cncf.io/wgs/platforms/maturity-model/v1/) that helps organizations evaluate the current state of their platform, understand the gaps, and inform the strategy for the future of their platform.One such direction that many high performing organizations share is treating your platform as a product. 
+
+“Nice!”, you say. “Alright then, give me the tools and methodologies to make my platform-as-a-product and then we’re good!”, you might add. But it’s not that easy. While there is a research initiative within the [CNCF Platform Working Group](https://tag-app-delivery.cncf.io/wgs/platforms/) to find out how organizations approach platform-as-a-product, one thing is already clear: It’s not just about a set of tools and methodologies, it’s about a mindset - Product Thinking. 
+
+But now what’s this Product Thinking? And why should you care? Product Thinking is making sure you build the right things without wasting time, resources and money to build something that users don’t care about. That’s why this blog is starting with a look at what Product Thinking is, how you can benefit from it and how it can make your platform into something engineers love to use.
+
+## What is product thinking?
+
+Product thinking is the process of identifying and prioritizing the problems your customers face and creating value by solving them.
+
+Notice that didn't include solutions or features.
+
+### Solving Problems vs Creating Solutions
+
+In order to have a successful product knowing **why** you are building something is just as important as **what** you build. After all, if you don't know why you are building something, how do you know you are building the right thing? Successful product teams understand this and focus their efforts on understanding user pain points before jumping in with solutions. 
+
+It is all too natural for engineers to jump into solutioning, but this approach has downsides. By jumping straight to solutioning, you cut off exploration of the problem, which could have a larger scope than you realize. The solution you create might not fit user needs and you've just wasted time and money building the wrong thing. Not only have you wasted your own time, but your users still struggle with the same underlying problem. Perhaps they even experience additional pain by being forced to adopt your ill-fitting solution.
+
+Product thinking helps you become proactive in learning about your users' problems and unearthing new opportunities to improve their experience. It gives you the ability to de-risk your effort by fully understanding the underlying problems and validating your assumptions about your potential solutions before you build.
```

Comment:
Here we have `Product thinking`. Some point later there is `product thinking` and most often we have `Product Thinking`. I think we should align on one way on how to caps that and I would say CamelCase with `Product Thinking` everywhere would be nicest because it's a name and all

Reply:
I've capitalized everywhere.